3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/fibonacci/src/lib.rs

```diff
@@ -1,10 +1,10 @@
 pub fn fib(n: u32) -> u32 {
-    if n <= 2 {
+    return if n <= 2 {
```

Comment:
마지막 문장에는 `return`을 생략할 수 있습니다.
```suggestion
    if n <= 2 {
```